python/ray/tune/suggest/skopt.py

```diff
@@ -39,19 +54,47 @@ class SkOptSearch(SuggestionAlgorithm):
         >>> }
         >>> algo = SkOptSearch(optimizer,
         >>>     ["width", "height"], max_concurrent=4,
-        >>>     reward_attr="neg_mean_loss")
+        >>>     reward_attr="neg_mean_loss", points_to_evaluate=current_best_params)
     """
 
     def __init__(self,
                  optimizer,
                  parameter_names,
                  max_concurrent=10,
                  reward_attr="episode_reward_mean",
+                 points_to_evaluate=None,
+                 evaluated_rewards=None,
                  **kwargs):
         assert skopt is not None, """skopt must be installed!
             You can install Skopt with the command:
             `pip install scikit-optimize`."""
         assert type(max_concurrent) is int and max_concurrent > 0
+        if points_to_evaluate is None:
+            points_to_evaluate = []
+        elif not isinstance(points_to_evaluate[0], (list, tuple)):
+            points_to_evaluate = [points_to_evaluate]
+        if not isinstance(points_to_evaluate, list):
+            raise ValueError(
+                "`points_to_evaluate` should be a list, but got %s" %
+                type(points_to_evaluate))
+        if isinstance(evaluated_rewards, Iterable):
+            evaluated_rewards = list(evaluated_rewards)
+        elif isinstance(evaluated_rewards, numbers.Number):
+            evaluated_rewards = [evaluated_rewards]
+        self._initial_points = []
+        if points_to_evaluate and evaluated_rewards:
```

Comment:
At this point, we can check if their lengths are the same. Then shouldn't we just loop through every point and reward and call optimizer.tell(point, reward)?